src/main/resources/META-INF/plugin.xml

```diff
@@ -1,28 +1,29 @@
 <idea-plugin url="https://github.com/gilday/dark-mode-sync-plugin">
-  <id>com.github.gilday.darkmode</id>
```

Comment:
Similar to my concern about moving class members within a file, let's avoid changing more than we have to here. I think you've alphabetized the XML elements, but I'd prefer to keep this changeset focused on what is necessary to introduce Windows support

Reply:
see above.